sirepo/util.py

```diff
@@ -185,7 +187,14 @@ def flask_app():
 
 def in_flask_request():
     import sys
+    global _logged_in_request, _logged_outside_request
     f = sys.modules.get('flask')
+    if _logged_in_request < 10 and f and f.request:
```

Comment:
I think it is important to note that this is a global that will be accessed in a threaded context. It's ok, but it just should be noted that the number "10" doesn't need to be exact.

I would this code this way:

```py
if not f:
    if _log_not_flask < 10:
        _log_not_flask += 1
        pkdlog('flask is not imported')
    return False
if not f.request:
    if _log_not_request < 10:
        _log_not_request += 1
        pkdlog('flask.request is False')
    return False
return True
```

Reply:
Thanks. I like that better. 7a3d2ae8